dist/openshift/cincinnati.yaml

```diff
@@ -208,8 +208,8 @@ objects:
       namespace: cincinnati
     spec:
       minAvailable: 1
-    selector:
-      app: cincinnati
+      selector:
```

Comment:
@vrutkovs looks like we forgot `matchLabels:`

Reply:
https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/zookeeper-pod-disruption-budget-minavailable.yaml